dotnet/src/Connectors/Connectors.Memory.SqlServer/SqlServerMemoryEntry.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Connectors.SqlServer;
+
+/// <summary>
+/// A SQL Server or Azure SQL memory entry.
+/// </summary>
+public record struct SqlServerMemoryEntry
+{
+    /// <summary>
+    /// Unique identifier of the memory entry.
+    /// </summary>
+    public string Key { get; set; }
+
+    /// <summary>
+    /// Attributes as a string.
+    /// </summary>
+    public string MetadataString { get; set; }
```

Comment:
I used the same name as in i.e. [SQLite](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Connectors/Connectors.Memory.Sqlite/Database.cs#L16) or [Postgres](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresMemoryEntry.cs#L21). But it can surely be changed.